source/operations/troubleshooting.rst

```diff
@@ -57,17 +57,18 @@ Registering Your MinIO Deployment with SUBNET
 
    .. tab-item:: Console Airgapped
 
-      You can register MinIO deployments for SUBNET even if they exists an airgap, firewall, or otherwise no direct Internet access.
+      You can register MinIO deployments for SUBNET even if they exist an airgap, firewall, or otherwise no direct Internet access.
```

Comment:
I'm having trouble parsing the intent here. 

"You can register ... deployments even if they are airgapped, firewalled, or otherwise have no direct Internet access:" ?

Reply:
reworked